src/.server/fund.ts

```diff
@@ -1,30 +1,49 @@
 import type * as endowDb from "@better-giving/endowment/db";
 import type { FundUpdate, SingleFund } from "@better-giving/fundraiser";
+import { type SlugEnvGsi, fundGsi } from "@better-giving/fundraiser/db";
 import type * as db from "@better-giving/fundraiser/db";
+import { fundId as fundIdSchema } from "@better-giving/fundraiser/schema";
 import { tables } from "@better-giving/types/list";
 import type { AttrNames } from "@better-giving/types/utils";
-import { BatchGetCommand, GetCommand, UpdateCommand, ap } from "./aws/db";
-import { dbUpdate } from "./aws/helpers";
+import * as v from "valibot";
+import { segment } from "../api/schema/segment";
+import {
+  BatchGetCommand,
+  QueryCommand,
+  type QueryCommandInput,
+  UpdateCommand,
+  ap,
+} from "./aws/db";
+import { buildProfileUpdateParams, dbUpdate } from "./aws/helpers";
 import { env } from "./env";
 
 export const getFund = async (
-  fundId: string
+  fundIdOrSlug: string
```

Comment:
nit: 
```suggestion
  uuidOrSlug: string
```
indicating that fund identifier is either `uuid` or `slug`

or just add comment

```
// uuid or slug
fundId: string
```
`fundIdOrSlug` seems like `fundId` is some id  technology per se i.e `fundIdOrSlugOrUuidv4OrNanoid`

Reply:
Addressed in https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/3579/commits/6d4aac4b28637c3f995af8a52020514953516260